### DISCUSSION TITLE
"error":"time: unknown unit \"ht\" in duration \"8760ht\""

### DISCUSSION THREAD

**Discussion by skreddy6673** [concourse/concourse#8452] [Help & Support]
Ran into the issue while upgrading from v6.5.1 to v7.0.0. The web nodes kept restarting.

{"timestamp":"2022-06-26T22:51:09.849256356Z","level":"error","source":"atc","message":"atc.list-all-resources.failed-to-get-all-visible-resources","data":{"cluster":"Concourse_Prod","error":"time: unknown unit \"ht\" in duration \"8760ht\"","session":"63"}}

Any idea how to identify where the `8760ht` is coming from?

**Comment by skreddy6673** [comment]
@taylorsilva Sorry to disturb you by pinging directly. Do you know how can i narrow it down to the pipeline that is having this config?

**Reply by taylorsilva** [reply]
More discussion on discord. For anyone searching and finding this thread, the problem is likely in the resources table, specifically the config column. If querying doesn't help you find it you can try using the fly cli and bash scripting to get all pipelines and then get the config for each pipeline. You should fail to get one pipeline, that's the bad one
